app/policies/account_policy.rb

```diff
@@ -1,6 +1,6 @@
 class AccountPolicy < ApplicationPolicy
   def create?
-    token.present? && token.scopes&.include?('account:write')
+    token && token.authorized?(ENV['PRX_ADMIN_ID'], :admin)
```

Comment:
I'm assuming this `PRX_ADMIN_ID` is the magical number 8 _user_ id? Might be nice to see `PRX_ADMIN_USER_ID` in there just so there's no confusion.

Reply:
Just to clarify, this means that regular (non-admin) users cannot create accounts using this policy-- just oauth authorized applications (like `id.prx.org`) will create accounts?

Reply:
Hmm... I'm not sure I like having any user_ids in CMS.  Or is this actually an account id?

So does the `account:write` scope not work?  I'd think a "normal" user/client wouldn't be able to ask for that.  ID would only issue that scope to itself, for specific purposes.

Reply:
Yeah, that's an account id. I don't like this either.

We could use scopes, to do that this ticket will require a fair amount of functionality added on top of the existing OAuth2 implementation. ID has no way to control which scopes it issues, beyond ensuring that they're on our whitelist. I think the proper way to implement this would be to add scopes as a property of client applications, and then grab them from the database to add them to the token.

I discussed this with Andrew and what we landed on is that since individual accounts can't be created without the ID of a user with no account, and all other account types require admin approval, this isn't worth adding substantial new functionality to our auth system. We just allow anyone with a valid token to create an individual account for a user who doesn't have one (i.e., the existing policy). Then, if we end up finding that there are other use cases for scopes that grant permissions, we can revisit.